Otherworldly Pines/Assets/Scripts/MovingPlatform.cs

```diff
@@ -65,16 +75,18 @@ private void PauseIfEndPoint()
         // endpoint pause if/else
         if (timer >= pauseDuration)
         {
-            float speedMult = targetPos.y - this.gameObject.transform.position.y;
-            transform.position = Vector3.MoveTowards(transform.position, targetPos, moveDist);
+            direction = (targetPos - startingPos).normalized;
+            rb.MovePosition(transform.position + direction*moveDist);
+            //float speedMult = targetPos.y - this.gameObject.transform.position.y;
+            //transform.position = Vector3.MoveTowards(transform.position, targetPos, moveDist);
         }
         else
         {
             timer += Time.deltaTime;
         }
     }
     // Update is called once per frame
-    void Update()
+    void FixedUpdate()
```

Comment:
I think we might still want to use update to track time, not sure though

Reply:
I checked and documentation says that Time.deltaTime will track time properly in FixedUpdate without needing Update to do so, so I think we're good on that front